front/app/modules/commercial/analytics/admin/hooks/usePostsFeedback/parse.ts

```diff
@@ -87,7 +88,7 @@ export const parseStackedBarsData = (
     statusRows.reduce(
       (acc, row) => ({
         ...acc,
-        [row['dimension_status.id']]: row.count,
+        [get(row, 'dimension_status.id')]: row.count,
```

Comment:
i understand why we have to use this method instead of accesing a object value with its key but i will cry a little bit every time i use it.

Reply:
Hahah yeah I feel you. Once we enable typescript strict mode we can get rid of this again, but right now there's still 1000s of errors when we do that

Reply:
ok i understand